### ISSUE TITLE
What is this file ?  rcindex 

### ISSUE THREAD

**Issue by mobilelifeful** [developeranaz/RCLONE-SERVE-INDEX#23]
Thank you for sharing this. 
Any change we can know what rcindex is ?

**Comment by developeranaz** [comment]
sorry about that file, i lost that uncompiled version. 
This file is recovered from one of my older docker image . That's why i hadn't released it officially.
I need to rebuilt that file again, be patient i'll rebuild it and release it soon.

And I'm not good at programming, I made it with the help of stackoverflow and other websites, that's why it's hard to build it again to me. 😅

**Comment by bolavefasfas** [comment]
> 

Brooo so you are saying this is a compiled version of a docker image right ??
I want to create such files how to create them 
any wiki page please

**Comment by developeranaz** [comment]
No, rcindex contains just simple rclone commands, i thought it would get rid of heroku ban at that time. Just use `rclone serve http yourremote:` or `rclone rcd --rc-serve`

**Comment by bolavefasfas** [comment]
Bro whats the difference between, How are they different from one other 
1. rclone serve http remote:
2. rclone serve webdav remote:
3. rclone serve ftp remote:
4. rclone rcd --rc-serve remote:
5. rclone rcd --rc-web-gui remote:

and which one can bypass 5 gb mega limit for index?

**Comment by bolavefasfas** [comment]
Thanks Doubts clarified 
so 
`rclone serve http/ftp/webdav remote:` is only for 1 remote
`rclone rcd --rc-serve` this is just an index for all remotes in config 
`rclone rcd --rc-web-gui` this is like a file browser and for copying or moving or seeing stuff 

Please let me know which commands were wrong ig 4,5 th after them remote wont come right ?

**Comment by developeranaz** [comment]
> 

1. rclone serve http remote:
2. rclone serve webdav remote:
3. rclone serve ftp remote:
4. rclone rcd --rc-serve 
5. rclone rcd --rc-web-gui
